samples/openshift-cloud-terminal.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: workspace.che.eclipse.org/v1alpha1
+kind: Workspace
+metadata:
+  name: openshift-terminal
+  annotations:
+    org.eclipse.che.workspace/immutable: "true"
```

Comment:
It's needed to make sure that nobody (who has edit rights to a workspace) is not able to modify workspace and put custom editor that will be like send personal user's token somewhere or just print it into logs.